Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM docker.io/library/golang:1.19.4-alpine3.17 as builder
+WORKDIR /go/src/github.com/kiagnose/kubevirt-rt-checkup
+COPY . .
+RUN go build -v -o ./bin/kubevirt-rt-checkup ./cmd/
+
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.1.0-1656.1669627757
```

Comment:
How about defining a default user for the checkup container?

Reply:
Done.